orc8r/gateway/go/services/bootstrapper/gateway_info/get_info.go

```diff
@@ -44,7 +45,22 @@ func GetFormatted() (string, error) {
 			config.GetMagmadConfigs().BootstrapConfig.ChallengeKey, err)
 		return res, err
 	}
-	res = fmt.Sprintf("%s\nChallenge Key:\n--------------\n%s\n",
-		res, base64.StdEncoding.EncodeToString(marshaledPubKey))
+	res = fmt.Sprintf("%s\nChallenge Key:\n--------------\n%s\n%s\n",
+		res, base64.StdEncoding.EncodeToString(marshaledPubKey), PrintBuildInfo())
```

Comment:
it's better to add delimiter() to magmad's main() - this is bootstrapping GW info function